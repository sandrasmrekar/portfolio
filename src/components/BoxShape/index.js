import styles from "./index.module.css";

export default function BoxShape({ children }) {
  return (
    <div className={styles.boxContainer}>
      <div className={styles.line} />
      <div className={styles.middle}>
        <div className={styles.center}>{children}</div>
      </div>
      <div className={styles.line} />
    </div>
  );
}
