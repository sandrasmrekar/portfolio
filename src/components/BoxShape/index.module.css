/* box */

.boxContainer {
  width: 50%;
}

.line {
  background-color: #e3e1ff;
  height: 10px;
}
.middle {
  justify-content: space-around;
  background-color: #131313;
  flex-direction: column;
  height: 380px;
  display: flex;
}

.center {
  text-align: center;
  color: white;
}

@media screen and (max-width: 600px) {
  .boxContainer {
    width: 100%;
  }
  .middle {
    height: 300px;
  }
}
