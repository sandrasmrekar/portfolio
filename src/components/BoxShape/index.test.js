import React from 'react';
import renderer from 'react-test-renderer';
import BoxShape from '.';

test('check the right rendering', () => {
    const tree = renderer.create(<BoxShape/>).toJSON();
    expect(tree).toMatchSnapshot();
})

