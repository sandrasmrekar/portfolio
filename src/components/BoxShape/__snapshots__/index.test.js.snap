// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check the right rendering 1`] = `
<div
  className="boxContainer"
>
  <div
    className="line"
  />
  <div
    className="middle"
  >
    <div
      className="center"
    />
  </div>
  <div
    className="line"
  />
</div>
`;
